### ISSUE TITLE
MongoError: not authorized on admin to execute command { serverStatus: 1 }

### ISSUE THREAD

**Issue by dnieuwstadt** [apostrophecms/apostrophe#1227]
After creating an instance on mLab and using env vars to connect the app to the database we recieve the following error every 10 seconds in the console:

```
name: 'MongoError',
message: 'not authorized on admin to execute command { serverStatus: 1 }',
ok: 0,
errmsg: 'not authorized on admin to execute command { serverStatus: 1 }',
code: 13,
codeName: 'Unauthorized'
```

The app successfully connects to the database and provisions the Apostrophe collections and we are able to launch the app and login but the console becomes inundated with the above error. We have had a browse through the source and the culprit seems to be the "keepalive" solution that has been put in place.

**Comment by boutell** [comment]
Hmmm, any query would do, we could plug in something else. The entire
mechanism might only be needed for a very narrow range of mongo versions.

**Comment by boutell** [comment]
There's a PR to fix this in review, it's in #1228

**Comment by boutell** [comment]
This shipped.
